modules/ROOT/pages/errors/gql-errors.adoc

```diff
@@ -1508,5 +1594,33 @@ Status description:: error: procedure exception - procedure sandboxed. `{ $sig }
 
 Status description:: error: procedure exception - procedure compilation failed. Failed to compile procedure/function defined in `{ $procClass }`: `{ $msg }`
 
+=== 52N36
+
+Status description:: error: procedure exception - invalid procedure argument with API documentation hint. Invalid argument `{ $value }` for `{ $procParam }` on procedure `{ $proc }`. The expected format of `{ $procParam }` is outlined in the `{ $procParamFmt }` API documentation.
+
+=== 52N37
+
+Status description:: error: procedure exception - procedure execution error. Execution of the procedure `{ $proc } ` failed.
+
+=== 52U00
+
+Status description:: error: procedure exception - custom procedure execution error cause. Execution of the procedure `{ $proc }` failed due to `{ $msgTitle }`: { $msg }
+
```

Comment:
Maybe it does not matter as you are anyway doing a refactoring in one of your other PRs but in this format we should start a new subsection for Function exception here

Reply:
You're right. I'll add it here as well.